priority-queue/PriorityQueueConsumerHigh/PriorityQueueConsumerHigh.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Because of the suggestion later in this file to move to 4.x, we can then move to .net 6 for this.

Reply:
Done for all projects

Reply:
Fancy, thanks for the update to 6